godot_project/editor/input_handlers/molecular_structures/add_posing_atom_input_handler.gd

```diff
@@ -353,7 +353,7 @@ func _is_shortcut_pressed() -> bool:
 ## Input handlers will execute _forward_input_* in an order dictated by this parameter
 ## highter priority value means the input handler will execute first
 func get_priority() -> int:
-	return BuiltinInputHandlerPriorities.ADD_ATOM_INPUT_HANDLER_PRIORITY
+	return BuiltinInputHandlerPriorities.ADD_POSING_ATOM_INPUT_HANDLER_PRIORITY
```

Comment:
was using the wrong priority, causing a warning about 2 input handlers using the same priority